modules/webxr/doc_classes/WebXRInterface.xml

```diff
@@ -285,5 +312,83 @@
 		<constant name="TARGET_RAY_MODE_SCREEN" value="3" enum="TargetRayMode">
 			Target ray from touch screen, mouse or other tactile input device.
 		</constant>
+		<constant name="HAND_JOINT_WRIST" value="0" enum="HandJoint">
+			Wrist joint.
+		</constant>
+		<constant name="HAND_JOINT_THUMB_METACARPAL" value="1" enum="HandJoint">
+			Thumb metacarpal joint.
+		</constant>
+		<constant name="HAND_JOINT_THUMB_PHALANX_PROXIMAL" value="2" enum="HandJoint">
+			Thumb phalanx proximal joint.
+		</constant>
+		<constant name="HAND_JOINT_THUMB_PHALANX_DISTAL" value="3" enum="HandJoint">
+			Thumb phalanx distal joint.
```

Comment:
All of these basically repeat the name of the constant. It would be nice to simplify their meaning taking advantage of the description's space. Currently I have no concrete suggestion for all of these.

E.g.
"Thumb metacarpal joint. The lowest joint of the thumb"(?)